Common/Securities/SecurityCache.cs

```diff
@@ -37,7 +37,7 @@ public class SecurityCache
         /// <summary>
         /// Gets the most recent price submitted to this cache
         /// </summary>
-        public decimal Price { get; private set; }
+        public decimal Price { get; set; }
```

Comment:
I think there is a set market price function on security that should be called to do this. If a property is private set, care should be taken when changing to public.